docs/azure/services/Azure_Functions.md

```diff
@@ -0,0 +1,109 @@
+
+Additionally, a “system-key” is used for webhook authentication required by different extensions used within a Function App. This system-key will be created by extension which will also determine whether it’s a host-level or function-level key.
+
+As with other resources, relying on shared keys to access a resource is not an ideal scenario in terms of secrets management across an estate. Under the Authentication blade, it is possible to set up an authentication provider such as Azure AD which makes use of the tenant's authentication capabilities to restrict access to function endpoints. Further information can be found [here](https://learn.microsoft.com/en-us/azure/app-service/overview-authentication-authorization).
+
+### IAM
+To manage Azure Functions either through portal or programmatically, a user must be assigned the relevant Azure role. Standard practices based on the zero trust model found [here](https://learn.microsoft.com/en-us/azure/active-directory/roles/best-practices) should be followed. To view which role assignments assigned to users that can interact with Azure Functions, the following command can be run:
+```CLI
+az role assignment list –scope /subscriptions/SUBSCRIPTION-ID/resourceGroups/myGroup/providers/ Microsoft.Web/sites /myFunction
+```
+
+### CORS Configuration
+CORS Access determines which web applications running in another domain can make requests to a Function’s trigger endpoint. Allowed Origins should be limited to a strict set of domains which should be able to make cross-origin requests. Allow-all (“*”) statements should be avoided.
+
+The following command can be run to show any allowed origins:
+```CLI
+az functionapp cors show --name MyFunctionApp --resource-group MyResourceGroup
+```
+### Logging and Monitoring
+In the Overview blade, the Microsoft Defender for Cloud service will provide recommendations and security alerts in relation to a function that has been evaluated. This can be a quick source of common misconfigurations and provides alerts for certain possibly malicious activities that had occurred. However, a more in-depth approach to reviewing the security configuration should be taken especially when considering networking restrictions which often depend on the context of the broader platform which would require manual review.
+
+Activity logs are a different source of telemetry for the resources that are enabled by default. All "/write" operations to Azure Functions resources will be logged, however this does not include "/read" operations.
```

Comment:
Although a useful clarification on general Azure Monitor behaviour can we clarify what type of log sources are available for Function Apps in Azure Montiro. AKA what sort of resource logs you can collect as a whole.